assets/stylesheets/components/_notification-banner.scss

```diff
@@ -15,6 +15,32 @@
   // scss-lint:enable SelectorFormat
 }
 
+.notification-banner__hiviz {
+  background-color: tint($yellow, 25%);
+
+  a {
```

Comment:
Re the second bit, that's a fair point, BUT what's happening is that colours are being tuned away form the base palette by _as little as possible_ in order to get to a correct contrast ratio. This is going to differ...